TypeScript.sublime-settings

```diff
@@ -5,5 +5,7 @@
 
     // empty string, or one of "region.redish", "region.orangish", "region.yellowish", "region.greenish", "region.bluish", "region.purplish", "region.pinkish"
     "error_color": "",
+    "error_icon": "",
```

Comment:
This means there's always a default, which means users haven't been seeing errors since we sent out the update last week.